### ISSUE TITLE
Module `AccessibilityInfo` does not exist in the Haste module map

### ISSUE THREAD

**Issue by myvary** [reactnativecn/react-native-pushy#230]
```
Bundling with React Native version:  0.57.0
Loading dependency graph, done.
Error: Unable to resolve module `AccessibilityInfo` from `/Users/liuxin/work/Android/******/node_modules/react-native/Libraries/react-native/react-native-implementation.js`: Module `AccessibilityInfo` does not exist in the Haste module map

This might be related to https://github.com/facebook/react-native/issues/4968
To resolve try the following:
  1. Clear watchman watches: `watchman watch-del-all`.
  2. Delete the `node_modules` folder: `rm -rf node_modules && npm install`.
  3. Reset Metro Bundler cache: `rm -rf /tmp/metro-bundler-cache-*` or `npm start -- --reset-cache`.
  4. Remove haste cache: `rm -rf /tmp/haste-map-react-native-packager-*`.
    at ModuleResolver.resolveDependency (/Users/liuxin/work/Android/*******/node_modules/metro/src/node-haste/DependencyGraph/ModuleResolution.js:209:1301)
    at ResolutionRequest.resolveDependency (/Users/liuxin/work/Android/******/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:83:16)
    at DependencyGraph.resolveDependency (/Users/liuxin/work/Android/******/node_modules/metro/src/node-haste/DependencyGraph.js:238:485)
    at Object.resolve (/Users/liuxin/work/Android/*******/node_modules/metro/src/lib/transformHelpers.js:180:25)
    at dependencies.map.result (/Users/liuxin/work/Android/******/node_modules/metro/src/DeltaBundler/traverseDependencies.js:311:29)
    at Array.map (<anonymous>)
    at resolveDependencies (/Users/liuxin/work/Android/******/node_modules/metro/src/DeltaBundler/traverseDependencies.js:307:16)
    at /Users/liuxin/work/Android/******/node_modules/metro/src/DeltaBundler/traverseDependencies.js:164:33
    at Generator.next (<anonymous>)
    at step (/Users/liuxin/work/Android/******/node_modules/metro/src/DeltaBundler/traverseDependencies.js:266:307)
```
查看Issues #224 ,有相同问题。我在这里斗胆再发一遍，orz

#### 目前只能通过降低react native来解决吗？

**Comment by myvary** [comment]
谢谢大佬们的pr，我目前用的是自建的code-push 。没有验证目前是否可用，暂时关闭

**Comment by Abreto** [comment]
> 谢谢大佬们的pr，我目前用的是自建的code-push 。没有验证目前是否可用，暂时关闭

我这边试了下可以正常热更新

(另外能问下你自建 code-push 参考的资料吗？以及是否支持 0.57 版本？

**Comment by myvary** [comment]
> > 谢谢大佬们的pr，我目前用的是自建的code-push 。没有验证目前是否可用，暂时关闭
> 
> 我这边试了下可以正常热更新
> 
> (另外能问下你自建 code-push 参考的资料吗？以及是否支持 0.57 版本？

[code push资料](https://github.com/crazycodeboy/RNStudyNotes/tree/master/React%20Native%E5%BA%94%E7%94%A8%E9%83%A8%E7%BD%B2%E3%80%81%E7%83%AD%E6%9B%B4%E6%96%B0-CodePush%E6%9C%80%E6%96%B0%E9%9B%86%E6%88%90%E6%80%BB%E7%BB%93#%E9%9B%86%E6%88%90codepush-sdk)
[具体方式](https://github.com/lisong/code-push-server/blob/master/docker/README.md)
